pkg/apply/event/event.go

```diff
@@ -154,3 +165,14 @@ type DeleteEvent struct {
 	Object     *unstructured.Unstructured
 	Error      error
 }
+
+type InventoryAddEvent struct {
```

Comment:
So what happens if the `InventoryAddEvent` fails? It seems like we should just bail out since it isn't safe to continue. So I'm uncertain what the use is for the `Error` property here. In general for the events, if there is a fatal error, we return an event of type `ErrorEvent`.

I'm thinking we should not add these events unless we have a use-case for it. We can always add it later if it needed.

Reply:
Removed inventory events and tests.